nservicebus/sqlserver/index.md

```diff
@@ -28,38 +28,63 @@ The SQL Server transport is a hybrid queueing system which is neither store-and-
  * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
  * Great tooling (SSMS)
  * Maximum throughput for any given endpoint is on par with MSMQ
+ * Free to start (Express edition)
  * Queues support competing consumers (multiple instances of same endpoint feeding off of same queue) so scale-out doesn't require using [distributor](/nservicebus/scalability-and-ha/distributor/)
 
 
 ### Disadvantages
 
  * No local store-and-forward mechanism, meaning that when SQL Server instance is down the endpoint cannot send nor receive messages
- * In centralized deployment maximum throughput applies for the whole system, not individual endpoints. If SQL Server on a given hardware can handle 2000 msg/s, each one of the 10 endpoints deployed on this machine can only receive 200 msg/s (on average).
+ * In centralized deployment maximum throughput applies for the whole system, not individual endpoints. If SQL Server on a given hardware can handle 2000 msg/s, each one of the 10 endpoints deployed on this machine can only receive 200 msg/s (on average)
+ * When empty queue gets polled periodically (every 1 second) by the receiver to detect new messages. This can lead to processing delays for rarely used endpoints  
 
+## Deployment considerations 
 
-## Usage Scenarios
+### Introduction
+Any process hosting NServiceBus operates and manages different types of data, those are:
+ * Business data - data managed by NServiceBus user, usually via code executed from inside message handlers
```

Comment:
I would not include sagas here. The intention was to point to data that is managed in terms of persistence (CRUD) by user written code. Does not apply to saga which are handled by NServiceBus